triage/render.js

```diff
@@ -113,10 +123,44 @@ function renderJobs(parent, buildsIterator) {
   var jobList = addElement(parent, 'ul');
   for (let [job, buildNumbersSet] of clusterJobs) {
     let buildNumbers = Array.from(buildNumbersSet).sort();
-    addBuildListItem(jobList, job, buildNumbers);
+    addBuildListItem(jobList, job, buildNumbers, counts[job]);
   }
 }
 
+function sparkLinePath(arr, width, height) {
```

Comment:
Please document this function. Also link to something that describes the SVG format and whatever a path is.

This is super cool but we cannot assume that your average JS engineer understands SVG